example_plugins/pair_plugin/pytest/test_example_pair.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) 2009-2022 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+from typing import List, Optional, Tuple
+import itertools
+import pytest
+
+import hoomd
+from hoomd.device import Device
+
+from hoomd import pair_plugin
+
+import numpy as np
+from numpy.typing import NDArray
+
+
+def build_binary_system(pos: Optional[List[float]] = None,
+                        device: Optional[Device] = None) -> hoomd.Simulation:
+    """Builds a two-particle system.
+
+    Builds a system of two particles, with one particle located at the origin
+    and the other located at `pos`. `pos` should be a list of 3 floats, and is
+    defaulted to [0.5, 0, 0]. The box is 4x4x4.
+    """
+    if pos is None:
+        pos = [0.5, 0, 0]
+    else:
+        if not isinstance(pos, list):
+            raise ValueError("`pos` must be a list")
+        if len(pos) != 3:
+            raise ValueError("`pos` must be a list of length 3")
+
+    if device is None:
+        device = hoomd.device.CPU()
+
+    sim = hoomd.Simulation(device, 0)
+
+    snap = hoomd.Snapshot()
+    snap.particles.N = 2
+    snap.particles.types = ["A"]
+    snap.particles.position[:] = [[0, 0, 0], pos]
+    snap.configuration.box = [4, 4, 4, 0, 0, 0]
+
+    sim.create_state_from_snapshot(snap)
+
+    return sim
+
+
+def harm_force_and_energy(
+        dx: List[float],
+        k: float,
+        sigma: float,
+        r_cut: float,
+        shift: Optional[bool] = False) -> Tuple[NDArray[np.float64], float]:
+
+    dr = np.linalg.norm(dx)
+
+    if dr >= r_cut:
+        return np.array([0.0, 0.0, 0.0], dtype=np.float64), 0.0
+
+    f = -k * (dr - sigma) * np.array(dx, dtype=np.float64) / dr
+    e = 0.5 * k * (dr - sigma)**2
+    if shift:
+        e -= 0.5 * k * (r_cut - sigma)**2
+
+    return f, e
+
+
+rng = np.random.default_rng(seed=0)
+positions = 0.5 * rng.random((10, 3)) - 0.25
+positions = list(positions) + list(2.0 * rng.random((10, 3)) - 1.0)
```

Comment:
Please set positions to an explicit list (can be copied and pasted output from this code). This will ensure that future changes in numpy don't cause this test to fail.

Reply:
I just replaced these with an `np.linspace` call.